rim/templates/rim/client_list.html

```diff
@@ -38,31 +38,32 @@ <h1>{% trans 'Client List' %}</h1>
             </tbody>
         </table>
     </div>
-    <div class='grid-container'>
-        <div class='grid-margin-x'>
-            <div class='grid-x'>
-                <div class='small-3 large-3 cell'>
-                </div>
-                <div class='small-6 large-6 cell'>
-                    {% if is_paginated %}
-                        <div class='page_nav'>
-                            {% if page_obj.has_previous %}
-                                <a class='page_link' href='?page={{ page_obj.previous_page_number }}&amp;{{ search_data.urlencode }}'>Previous</a> |
-                            {% endif %}
-                            Page {{ page_obj.number }}
-                            {% if page_obj.has_next %}
-                                 | <a class='page_link' href='?page={{ page_obj.next_page_number }}&amp;{{ search_data.urlencode }}'>Next</a>
-                            {% endif %}
+    <div class='btm-bar-wrapper'>
+        <div class='expanded align-center nav btm-bar'>
+            <div class='grid-margin-x'>
+                <div class='grid-x'>
+                    <div class='small-3 large-3 cell'></div>
+                    <div class='small-6 large-6 cell'>
+                        {% if is_paginated %}
+                            <div class='page_nav'>
+                                {% if page_obj.has_previous %}
+                                    <a class='page_link' href='?page={{ page_obj.previous_page_number }}&amp;{{ search_data.urlencode }}&amp;order={% if current_order_desc %}-{% endif %}{{ current_ordering_name }}'>Previous</a> |
```

Comment:
`search_data` is a string and doesn't have any `urlencode` method. So this won't work.
My vote is to turn `search_data` into a [QueryDict object](https://docs.djangoproject.com/en/2.2/ref/request-response/#querydict-objects) in the view.